.github/workflows/triage.yaml

```
@@ -1,50 +1,26 @@
 name: Triage
 permissions:
+  contents: read
   pull-requests: write
-"on":
-  pull_request:
-    types:
-      - opened
-      - synchronize
+on:
+  schedule:
+    - cron: "0 0 * * */0"
```

actionlint (1.7.12)

[error] invalid CRON format "0 0 * * */0" in schedule event: step of range should be a positive number: */0

(events)